### DISCUSSION TITLE
change image that shows with hyperlink

### DISCUSSION THREAD

**Discussion by noyloza** [bootscore/bootscore#323] [Q&A]
I am trying to change the bootscore logo that gets shown when I post the hyperlink in social media and messages. How can I change this to a custom image?

**Comment by crftwrk** [comment]
Social media usually fetches the featured image like here https://twitter.com/_bootscore/status/1568199073135529984.

If page or post doesn’t have a featured image then the first image of the site is used and of course, first image is the site logo. So, if you want to show a custom image on social media, simply add a featured image to your page/post.
